grr/proto/grr_response_proto/osquery.proto

```diff
@@ -17,6 +20,13 @@ message OsqueryArgs {
   optional string query = 1;
   optional uint64 timeout_millis = 2;
   optional bool ignore_stderr_errors = 3;
+  repeated string file_collect_columns = 4;
+}
+
+message OsqueryActionArgs {
```

Comment:
Unfortunately, this is not backward-compatible. It is stupid and I argued multiple times to change it because it breaks one of the fundamental rules of Protocol Buffers, but clients validate names of massages. So, renaming action arguments is not an option. For flow there is no such constraint, though.